topics/appendix/tools/tools_gradle_intellij_plugin.md

```diff
@@ -0,0 +1,1210 @@
+[//]: # (title: Gradle IntelliJ Plugin)
+
+<!-- Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+The Gradle IntelliJ Plugin is a plugin for the Gradle build system to help configuring your environment for building and testing plugins for IntelliJ-based IDEs.
+
+> Current Gradle IntelliJ Plugin version is [![GitHub Release](https://img.shields.io/github/release/jetbrains/gradle-intellij-plugin.svg?style=flat-square)](https://github.com/jetbrains/gradle-intellij-plugin/releases)
+>
+{type="note"}
+
+
+When upgrading to `1.x` version, please make sure to follow [migration guide](https://lp.jetbrains.com/gradle-intellij-plugin) to adjust your existing build script.
+
+This plugin allows you to build plugins for IntelliJ Platform using specified IntelliJ SDK and bundled or third-party plugins.
+
+The plugin adds extra IntelliJ-specific dependencies, patches `processResources` tasks to fill some tags (name, version) in `plugin.xml` with appropriate values, patches compile tasks to instrument code with nullability assertions and forms classes made with IntelliJ GUI Designer and provides some build steps which might be helpful while developing plugins for IntelliJ platform.
+
+> Gradle JVM should be set to `Java 11` (see <path>Settings/Preferences | Build, Execution</path>, <path>Deployment | Build Tools | Gradle</path>)
+>
+{type="tip"}
+
+> This project requires `Gradle 6.7` or newer, however it is recommended to use the latest Gradle available. Update it with:
+> ```Bash
+> ./gradlew wrapper --gradle-version=VERSION
+> ```
+>
+{type="tip"}
+
+
+## Usage
+To enable this plugin in your Gradle-based project, add the following entry to the `plugins` section:
+
+<tabs>
+<tab title="Kotlin">
+
+```kotlin
+plugins {
+    id("org.jetbrains.intellij") version "..."
+}
+```
+
+</tab>
+<tab title="Groovy">
+
+```groovy
+plugins {
+    id "org.jetbrains.intellij" version "..."
+}
+```
+
+</tab>
+</tabs>
+
+### Snapshot Release
+
+The Snapshot release is a pre-release version built nightly from the latest main branch.
+To use it, it is required to point Gradle to the dedicated snapshot repository by adding an entry to the Gradle settings file.
+
+> Current Gradle IntelliJ Plugin Snapshot version is [![GitHub Snapshot Release](https://img.shields.io/nexus/s/org.jetbrains.intellij/org.jetbrains.intellij.gradle.plugin?server=https%3A%2F%2Foss.sonatype.org)](https://github.com/jetbrains/gradle-intellij-plugin/releases)
```

Comment:
This happens because Stardust adds the `?_ijt=...` token parameter to the image URL, which already has `?..=..`, making the URL malformed. It'll work in the production.

```
https://img.shields.io/nexus/s/org.jetbrains.intellij/org.jetbrains.intellij.gradle.plugin?server=https://oss.sonatype.org?_ijt=u6ec15960daq7hll52h9e48psr
```